fw/strategy.go

```diff
@@ -44,14 +54,17 @@ type StrategyBase struct {
 
 // NewStrategyBase is a helper that allows specific strategies to initialize the base.
 func (s *StrategyBase) NewStrategyBase(
-	fwThread *Thread, strategyName enc.Component, version uint64, strategyLogName string,
+	fwThread *Thread,
+	strategyName enc.Component,
+	version uint64,
+	strategyLogName string,
 ) {
 	var err error
 	s.thread = fwThread
 	s.threadID = s.thread.threadID
 	s.name, err = enc.NameFromStr(StrategyPrefix)
 	if err != nil {
-		core.LogFatal(s, "StrategyPrefix is a bad NDN Name")
+		panic("StrategyPrefix is a bad NDN Name")
```

Comment:
Log should be used whenever possible. If necessary let's put it in a separate package.

Reply:
I feel for fatal logs panic is more appropriate to signal the developers intention (I don't "know" if log fatal would crash).

Reply:
`LogFatal` will call `panic` and kill the program.

Reply:
Yes, but visually it's more obvious to the developer when you see a panic -- means a crash
Also tooling is better at figuring that out